source/process/working-at-mattermost.rst

```diff
@@ -0,0 +1,73 @@
+==============================================================
+Working at Mattermost
+==============================================================
+
+The mission of Mattermost, Inc. is to give all organizations the power to achieve their highest priorities through modern communication. Our vision is a world where people are in constant, seamless communication about what matters most to them.
+
+Our company maintains the Mattermost open source project, providing easy-to-use, multi-lingual, self-hosted team communication across web, PC and phones. 
+
+We also provide Mattermost Enterprise Edition, a secure, modern, enterprise messaging solution for large organizations, as a commercial, fully supported extension of our open source software. 
+
+While our headquarters is in Palo Alto, California, we use Mattermost software to work remotely around the world. 
+
+Top 10 reasons to work for Mattermost: 
+
+1. **Join an outstanding team:** Work with thoughtful, talented, high performance people who care about results. 
+2. **Growth:** Join a growing open source project, a growing business, and a growing company. When you're growing, every day is your best day ever.
+3. **Work remote:** No commute, plus complete control of your environment and freedom to travel.
+4. **Do good:** Build open source software benefiting thousands of companies, research institutions, schools and non-profits.
+5. **Improve a product you use every day:** Your opinion matters. 
+6. **Be recognized:** Working in open source, your contributions are publicly visible and will echo on throughout your career. 
+7. **Focus on results:** Output matters, not hours worked. Enjoy life and don't burn out. 
+8. **Continual development:** Cross-training and rotation builds up everyone on the team. It's core to our culture. 
+9. **Global perspective:** Develop outstanding software in a multi-lingual, multi-cultural community. 
+10. **Friends around the world:** Build relationships with staff and contributors in dozens of countries. 
+
+----------------------------------------------------------
+Working Remotely 
+----------------------------------------------------------
+
+While we're headquartered in Palo Alto, California, like many open source companies, a large portion of Mattermost, Inc. works remotely from around the world. 
```

Comment:
Sentence is ambiguous (does "like many open source companies" apply to the Palo Alto or working remotely). 

Propose something like "We're headquartered in Palo Alto, California, but like many open source companies a large portion of Mattermost, Inc. works remotely from around the world."